doc/architecture/20170816-audit-plan-to-model-audit-data.md

```diff
@@ -0,0 +1,29 @@
+
+
+## Term definitions
+
+A **theme** is a high level grouping criteria for all the content that GOV.Uk is currently hosting.
+
+An **inventory plan** is a set of rules that define the content that belongs to a theme. 
+
+The **inventory sheet** refines an inventory plan by excluding formats by adding/removing sets of content items.
+
+**Audit Sheet**: is what a the department is going to audit. It is the inventory plan refined twice.
```

Comment:
I don't think "refined twice" is a good definition. Refined how? I think the Audit Sheet should be the subset of a Theme that we wish to audit (see process outlined above).